Adyen/UI/View Controllers/AddressLookup/Search/AddressLookupSearchViewController+EmptyView.swift

```diff
@@ -0,0 +1,128 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+extension AddressLookupSearchViewController {
+    
+    /// The view that is shown when the address lookup search result is empty
+    internal class EmptyView: UIView, SearchResultsEmptyView {
+        
+        private let style: AddressLookupSearchEmptyStyle
+        private let localizationParameters: LocalizationParameters?
+        private let dismissHandler: () -> Void
+        
+        public var searchTerm: String {
+            didSet { updateLabels() }
+        }
+        
+        private lazy var titleLabel: UILabel = {
+            let titleLabel = UILabel()
+            titleLabel.numberOfLines = 0
+            titleLabel.textAlignment = .center
+            titleLabel.translatesAutoresizingMaskIntoConstraints = false
+            titleLabel.setContentHuggingPriority(.defaultHigh, for: .vertical)
+            return titleLabel
+        }()
+        
+        private lazy var subtitleLabel: LinkTextView = {
+            let textView = LinkTextView { [weak self] _ in
+                self?.dismissHandler()
+            }
+            textView.accessibilityIdentifier = ViewIdentifierBuilder.build(scopeInstance: self, postfix: "textView")
+            return textView
+        }()
+        
+        /// Initializes the `EmptyView` for the ``AddressLookupSearchViewController``
+        ///
+        /// - Parameters:
+        ///   - searchTerm: The search term that caused an empty state.
+        ///   - style: The style of the view.
+        ///   - localizationParameters: The localization parameters.
+        ///   - dismissHandler: A closure that is called when manual entry is requested.
+        internal init(
+            searchTerm: String = "",
+            style: AddressLookupSearchEmptyStyle = .init(),
+            localizationParameters: LocalizationParameters? = nil,
+            dismissHandler: @escaping () -> Void
+        ) {
+            self.style = style
+            self.localizationParameters = localizationParameters
+            self.searchTerm = searchTerm
+            self.dismissHandler = dismissHandler
+            
+            super.init(frame: .zero)
+            
+            setupContent()
+            updateLabels()
+        }
+        
+        @available(*, unavailable)
+        internal required init(coder: NSCoder) {
+            fatalError("init(coder:) has not been implemented")
+        }
+    }
+}
+
+// MARK: - Updating Interface
+
+private extension AddressLookupSearchViewController.EmptyView {
+    
+    func setupContent() {
```

Comment:
small thing but this one is identical to search controllers `setupContent()`. Was thinking if we should make a common protocol/func for it

Reply:
Great catch! The views are actually so similar that I'll create a generic version of it 👌

Reply:
Fixed in 1f2519998a83402396d39b279bf7e24a717a8213